package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apisix-dashboard",
-  "version": "1.5.4",
+  "version": "1.6.1",
```

Comment:
will be updated to 2.0.0 once the rest works are done.

- [ ] full frontend's test
- [ ] update deployment docs
- [ ] issues under 1.6 milestone